src/js/azdata/azdataQueryPlan.js

```diff
@@ -302,20 +302,44 @@ azdataQueryPlan.prototype.init = function (container, iconPaths, badgeIconPaths)
     graph.convertValueToString = function (cell) {
         if (cell.value != null && cell.value.label != null) {
             let hasWindowsEOL = cell.value.label.includes('\r\n');
+            const joinStrings = (strArray) => {
+                if (hasWindowsEOL) {
+                    return strArray.join('\r\n');
+                }
+                else {
+                    return strArray.join('\n');
+                }
+            };
+
+
             let splitLabel = cell.value.label.split(/\r\n|\n/);
-            let cellLabel = splitLabel.map(str => {
+            let cellLabel = splitLabel.map((str, index) => {
                 let label = '';
-                label += str;
+
+                if (index === 0) {
+                    label += str.replace(/\(([^)]+)\)/g, '');
```

Comment:
Please add a comment and example of what this does. You can use the same one from your PR comment.